_includes/anchor_headings.html

```diff
@@ -92,6 +92,6 @@
         {{ include.bodySuffix }}
       </h{{ _workspace | last }}
     {% endcapture %}
-    {% capture edited_headings %}{{ edited_headings }}{{ new_heading | replace: '  ', '' | replace: new_line, '' }}{{ new_line }}{% endcapture %}
+    {% capture edited_headings %}{{ edited_headings }}{{ new_heading }}{{ new_line }}{% endcapture %}
   {% endfor %}
-{% endcapture %}{% assign headingsWorkspace = '' %}{{ edited_headings | strip }}
\ No newline at end of file
+{% endcapture %}{% assign headingsWorkspace = '' %}{{ edited_headings | strip }}
```

Comment:
Odd request, I know. But could you leave the lack of newline? It's [intentional](https://github.com/allejo/jekyll-anchor-headings/pull/4#issuecomment-439653390).